pkg/probe/log_fortianalyzer.go

```diff
@@ -0,0 +1,55 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type LogAnaResults struct {
+	Registration string `json:"registration"`
+	Connection   string `json:"connection"`
+	Received     int    `json:"received"`
```

Comment:
```suggestion
	Received     float64    `json:"received"`
```
Let the parser directly convert to float64